packages/suite/src/components/suite/banners/Banner.tsx

```diff
@@ -1,36 +1,38 @@
 import { ReactNode } from 'react';
-import styled, { useTheme } from 'styled-components';
-import { Button, Icon, SuiteThemeColors, variables } from '@trezor/components';
+import styled, { DefaultTheme, useTheme } from 'styled-components';
+import { Button, Icon, variables } from '@trezor/components';
+import { spacingsPx } from '@trezor/theme';
 
-const getBgColor = (variant: BannerProps['variant'], theme: SuiteThemeColors) => {
+const getBgColor = (variant: BannerProps['variant'], theme: DefaultTheme) => {
     switch (variant) {
         case 'info':
-            return theme.TYPE_BLUE;
+            return theme.backgroundAlertBlueBold;
```

Comment:
I feel like that those BGs being bold might be out of place considering the current stylistic direction. Waiting for an answer from Ondra!

![image](https://github.com/trezor/trezor-suite/assets/45338719/f7fc1482-1f8b-44e8-8873-92148787f81d)

Reply:
It follows previous design. Anyway, I think there should be one component, called `Banner` (or whatever) and it should solve both variants (currently `NotificationCard` and `Banner`). Something like in Google material design: https://mui.com/material-ui/react-alert/

`variant: standard`:
<img width="785" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/ef9bf4b4-809b-43d0-82e7-12f191cfa328">

`variant: filled`:
<img width="786" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/176bb375-28a4-41bd-ab00-b05da5d4ebe4">


`MUI API`
<img width="168" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/35855056-e579-4c73-8098-5a842e580540">